### DISCUSSION TITLE
Looking for command for Thermo ON/OFF in RT-ext mode

### DISCUSSION THREAD

**Discussion by buuhsmead** [Arnold-n/P1P2MQTT#118] [Q&A] [answered]
Did somebody find the command to 'Thermo ON/OFF' [A.2.2.4 / C-05] == 1 when the HP is in RT-ext mode. 
Tried several combinations, but no luck so far.

| mode | command | |
|------|-----|-|
| RT    | E35002D01 35003101 | Heating ON / RT Control ON |
| LWT | E35002F01 35003101 | Heating ON / RT Control ON |
| RT-ext | ???? |

And yes it is related to #115 

Thank you

**Comment by Arnold-n** [comment]
I haven't tried that mode yet myself - does it react to the on/off button on the wall room controller? If so, a log of P1P2/R/# (with J-mode set accordingly to output P1P2/R, i.e., `J3`) could be helpful to see which parameter is used.

**Comment by buuhsmead** [comment]
[r.log](https://github.com/user-attachments/files/16357883/r.log) The output log via J3 and 'P1P2/R/#'.

Current settings, manual via wall room controller :
| # | code | state |
|---|---|---|
| A224 | C-05 | 1 - Thermo |
|  A3111 | - | 1 - weather depend |
| A217 | C-07 | 1 - RT-ext |
| A21B | - | at Unit , instead of Room controller |
| A219 | F-0D | 2 - request |

Pushed the on/off button at about the following times, started in ON
| time | state |
|---|---|
| 080547  | OFF |
| 080624 | ON |
| 080704 | OFF |
| 080734 | ON |

**Reply by buuhsmead** [reply, marked as answer]
Hoi @Arnold-n added a log. Would be nice if you could have a look.

**Reply by Arnold-n** [reply]
I do not think this log shows anything different than when operating in LWT mode:
-OFF results in E35/002F set to 00 and E35/0031 set to 00
-ON results in E35/002F set to 01 and E35/0031 set to 01
So I am surprised that the commands `E35002F01 35003101`/`E35002F00 35003100` do not work.

**Comment by buuhsmead** [comment]
Thanks for looking into this.
I go either for LWT or RT mode and forget the RT-ext mode.
Tried several different cmds, but none of them worked.
And I don't want to brick the system.
